opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -18,7 +18,19 @@
 /**
  * Tracer is a simple, thin interface for Span creation and propagation across arbitrary transports.
  */
-public interface Tracer extends ActiveSpanSource {
+public interface Tracer {
+
+    /**
+     * @return the current {@link ScopeManager}, which may be a noop but may not be null.
+     */
+    ScopeManager scopeManager();
+
+    /**
+     * Set a new {@link ScopeManager}, presumably as part of initialization. The state of any already active
+     * {@link Scope} instance after this is only defined to do no harm to the process (i.e., not-crash). Again, this is
+     * only intended for use during initialization.
+     */
+    void setScopeManager(ScopeManager scopeManager);
```

Comment:
too sketchy to have to comment about race conditions... this can die, and in a separate PR we could introduce something like a `TracerBuilder` concept which would have a ScopeManager, and also perhaps an initial set of tags that are applied to all spans coming from that Tracer, etc.